testsuite/tests/test/crate-init/test.yaml

```diff
@@ -0,0 +1,4 @@
+driver: python-script
+indexes:
+    basic_index: 
+        in_fixtures: true
```

Comment:
This test seems to need no index, so it should use a configuration similar to the example in `testsuite/skels/no-index`